lib/util.js

```diff
@@ -331,12 +332,28 @@ function getSystemErrorName(err) {
   return internalErrorName(err);
 }
 
+/**
+ * @param {string} format
+ * @param {string} text
+ * @returns {string}
+ */
+ function colorText(format, text) {
+  validateString(format, 'format');
+  validateString(text, 'text');
+  const formatCodes = inspect.colors[format];
```

Comment:
If this is going to be public API, we should probably provide some additional validation for supported `format`s.

Reply:
> additional validation 
like?

Reply:
I was thinking we could validate that `format` is actually something supported by `inspect.colors`, but feel free to ignore if you want.

Reply:
If they pass anything apart as of now, we are just returning the text as in, do you feel it makes sense to check if the `format` is supported by `inspect.colors` and throw an error for non-supported inputs?

Reply:
Yes, that's what I originally meant.

Reply:
Is `inspect.colors` mutable and exposed to users?

Reply:
^ yes!